packages/rspeedy/plugin-react/src/entry.ts

```diff
@@ -198,6 +198,10 @@ export function applyEntry(
     })
 
     if (isLynx) {
+      const inlineScripts = typeof config.output?.inlineScripts === 'boolean'
```

Comment:
I think instead of `config.output`, we should use `   environment.config.output.inlineScripts`

Consider the following `lynx.config.js`:

```js
export default {
  environments: {
    lynx: { output: { inlineScripts: false } },
    web: { output: { inlineScripts: true } },
  },
}
```

You would get `true` when using `config.output.inlineScripts`, but `false` is expected.